src/VisualStudio/Setup.Next/source.extension.vsixmanifest

```diff
@@ -11,6 +11,11 @@
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Next.dll" />
+    </Actions>
+  </Installer>
   <Dependencies>
```

Comment:
But looks like Setup.Next does contain Microsoft.CodeAnalysis.EditorFeatures.Next.dll. Does that mean that the `<Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Next.dll" />` line added above is not required?